cql3/single_column_relation.cc

```diff
@@ -70,12 +70,16 @@ single_column_relation::new_EQ_restriction(database& db, schema_ptr schema, vari
     const column_definition& column_def = to_column_definition(schema, _entity);
     if (!_map_key) {
         auto term = to_term(to_receivers(schema, column_def), _value, db, schema->ks_name(), bound_names);
-        return ::make_shared<single_column_restriction::EQ>(column_def, std::move(term));
+        auto restr = ::make_shared<single_column_restriction::EQ>(column_def, term);
+        restr->rn.reset(new wip::oper(wip::idents{&column_def}, _relation_type, {term}));
+        return restr;
```

Comment:
Please use std::make_unique. Naked new is a flag that something tricky is going on, and nothing here should be tricky.

Reply:
Done.